Assets/Scripts/Manager/GameManager.cs

```diff
@@ -18,7 +18,7 @@ public class GameManager : MonoBehaviour {
 	public bool[] secretItemFound;
 
 	public bool firstRun;
-	public bool[] textTriggered;
+	public bool[] objectTriggered;
```

Comment:
just a suggestion but do you think we could do a map of object name -> bool instead of an array of bool.

I know that the index is saved in the checkOneTimeTrigger but I think it might be confusing if there is a bug or something to know what has been checked if it has been triggered or not.